sshfs.c

```diff
@@ -2338,22 +2382,9 @@ static int sshfs_readdir(const char *path, void *dbuf, fuse_fill_dir_t filler,
 		err = sftp_readdir_async(handle->conn, &handle->buf, dbuf,
 					 offset, filler);
 
-	return err;
-}
-
-static int sshfs_releasedir(const char *path, struct fuse_file_info *fi)
```

Comment:
Could you please move this function to the original location, so that the diff is easier to read?